server/utils/auth.js

```diff
@@ -24,9 +24,9 @@ export function setProfileFromGithub(
   },
   {
     id: githubId,
-    'avatar_url': picture,
+    avatar_url: picture,
     email: githubEmail,
```

Comment:
Is there a version that would require props with hyphens and underscores to be quoted?